config/testdata/gh1204_credentials

```diff
@@ -0,0 +1,9 @@
+[A]
```

Comment:
Could you also add testcases for:
1. duplicate profiles with same case type within a single file. 
2. similar profiles with different case type (that match if normalized) within a single file.
3. duplicate profiles with same case type across config & cred file. 
4. similar profiles with different case type (that match if normalized) across config & cred file.